### ISSUE TITLE
Makefile error on Colab happened recently (was working before) 

### ISSUE THREAD

**Issue by Arrrrthur** [AlexeyAB/darknet#4294]
Hi,
 I've been using your YOLOv3 on Colab since July and it works with this Makefile configuration:
`GPU=1
CUDNN=1
CUDNN_HALF=0
OPENCV=1
AVX=0
OPENMP=0
LIBSO=0
ZED_CAMERA=0`

However, when I tried to run it recently, there's an error with "make"

g++ -std=c++11 -Iinclude/ -I3rdparty/stb/include -DOPENCV `pkg-config --cflags opencv` -DGPU -I/usr/local/cuda/include/ -DCUDNN -Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -Ofast -DOPENCV -DGPU -DCUDNN -I/usr/local/cudnn/include obj/image_opencv.o obj/http_stream.o obj/gemm.o obj/utils.o obj/dark_cuda.o obj/convolutional_layer.o obj/list.o obj/image.o obj/activations.o obj/im2col.o obj/col2im.o obj/blas.o obj/crop_layer.o obj/dropout_layer.o obj/maxpool_layer.o obj/softmax_layer.o obj/data.o obj/matrix.o obj/network.o obj/connected_layer.o obj/cost_layer.o obj/parser.o obj/option_list.o obj/darknet.o obj/detection_layer.o obj/captcha.o obj/route_layer.o obj/writing.o obj/box.o obj/nightmare.o obj/normalization_layer.o obj/avgpool_layer.o obj/coco.o obj/dice.o obj/yolo.o obj/detector.o obj/layer.o obj/compare.o obj/classifier.o obj/local_layer.o obj/swag.o obj/shortcut_layer.o obj/activation_layer.o obj/rnn_layer.o obj/gru_layer.o obj/rnn.o obj/rnn_vid.o obj/crnn_layer.o obj/demo.o obj/tag.o obj/cifar.o obj/go.o obj/batchnorm_layer.o obj/art.o obj/region_layer.o obj/reorg_layer.o obj/reorg_old_layer.o obj/super.o obj/voxel.o obj/tree.o obj/yolo_layer.o obj/upsample_layer.o obj/lstm_layer.o obj/conv_lstm_layer.o obj/scale_channels_layer.o obj/sam_layer.o obj/convolutional_kernels.o obj/activation_kernels.o obj/im2col_kernels.o obj/col2im_kernels.o obj/blas_kernels.o obj/crop_layer_kernels.o obj/dropout_layer_kernels.o obj/maxpool_layer_kernels.o obj/network_kernels.o obj/avgpool_layer_kernels.o -o darknet -lm -pthread `pkg-config --libs opencv` -L/usr/local/cuda/lib64 -lcuda -lcudart -lcublas -lcurand -L/usr/local/cudnn/lib64 -lcudnn -lstdc++ 
obj/network.o: In function `resize_network':
network.c:(.text+0x1cc7): undefined reference to `resize_gaussian_yolo_layer'
obj/network.o: In function `num_detections':
network.c:(.text+0x2327): undefined reference to `gaussian_yolo_num_detections'
obj/network.o: In function `fill_network_boxes':
network.c:(.text+0x2b0f): undefined reference to `get_gaussian_yolo_detections'
obj/parser.o: In function `parse_gaussian_yolo':
parser.c:(.text+0x23e5): undefined reference to `make_gaussian_yolo_layer'
collect2: error: ld returned 1 exit status
Makefile:143: recipe for target 'darknet' failed
make: *** [darknet] Error 1

I tried Makefile configuration with OPENCV = 1 and 0 but neither works. Could you help me with this? Thank you

**Comment by AlexeyAB** [comment]
It seems that you downloaded new c-files, but didn't download new Makefile.

In your Makefile must be `gaussian_yolo_layer.o`
https://github.com/AlexeyAB/darknet/blob/master/Makefile#L121

**Comment by Arrrrthur** [comment]
Thanks! The problem is solved. 
I got all the file by directly cloning the repository
` !git clone https://github.com/AlexeyAB/darknet.git`

So I'm guessing it might be a version problem.
